project/Aliases.scala

```diff
@@ -0,0 +1,9 @@
+import sbt._
+
+object Aliases {
+
+  lazy val defineCommandAliases = {
+    addCommandAlias("ciBuild", ";clean; test") ++
+      addCommandAlias("ciRelease", ";clean; release with-defaults")
```

Comment:
I think Travis could use `ciBuild` for normal builds.  We can run `ciRelease` somehow (TBD) to do a release build.